src/style.css

```diff
@@ -12,68 +13,101 @@
 :root {
   --border-color: 1px solid #3f3f3f;
   --grey-color: #e5e5e5;
+  --text-grey: #4b4b4b;
 }
 
 body {
   font-family: Open Sans, sans-serif;
-  background: #dfdfdf;
+}
+
+.game-wrapper {
+  background: url(./images/Car-Race2.png);
```

Comment:
- [ ] Good job with the final touches, I think you accidentally used incorrect name for your image file and currently no styles are applied and we are getting this error :point_down: 
![image](https://user-images.githubusercontent.com/97521557/209406382-98da8962-5504-43fb-baca-fb4520f74137.png)

Please fix your image file name and make sure the webpack compiles the stylesheet successfully :+1: